anticheat/examples/bad_eval.ts

```
@@ -0,0 +1 @@
+eval("tsts");
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)